main/glossary/README.md

```diff
@@ -309,6 +307,14 @@ its asset type is valid.
 For more information, see the [ERTP Guide's Issuer section](/ertp/guide/issuers-and-mints.md)
 and the [ERTP API's Issuer section](/ertp/api/issuer.md).
 
+## Key
+
+A *key* is a [passable](#passable) containing no promises or errors, and can
+thus be synchronously compared for structural equivalence with another piece of data.
+If either side of the comparison contains promises and/or errors, equality is indeterminable.
+If both are fulfilled down to [presences](#presence) and local state, then either they're the
+same all the way down, or they represent different objects.
```

Comment:
Keys can be used as elements of CopySets and CopyBags, and as keys of CopyMaps. Amounts can only contain Keys.